modules/ocf_hpc/files/add_slurm_users

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+'''
+Script that adds users in the 'ocfhpc' LDAP group to SLURM as a SLURM user.
+'''
+import subprocess
+
+from ocflib.account import search
+from ocflib.account import utils
+
+DEFAULT_ACCOUNT = 'users'
+
+
+def main():
+    ocfhpc_ldap_user_set = set(utils.list_group('ocfhpc'))
+
+    # This produces a newline-separated list of SLURM users.
+    ocfhpc_slurm_user_list = subprocess.check_output(
+        ('sacctmgr', '-noP', 'list', 'users', 'format=User'),
+        universal_newlines=True,
+    ).split('\n')[:-1]
+    ocfhpc_slurm_user_set = set(ocfhpc_slurm_user_list)
+
+    users_to_be_added = ocfhpc_ldap_user_set.difference(ocfhpc_slurm_user_set)
+
+    if not users_to_be_added:
+        return
+
+    for new_user in users_to_be_added:
+        if not search.user_exists(new_user):
+            print('User {} in the HPC LDAP group does not exist.'.format(new_user))
```

Comment:
I'd go for a more meaningful error, such as:

"User {} in the HPC LDAP group not managed by slurm."